src/libxrpl/protocol/STParsedJSON.cpp

```diff
@@ -647,8 +647,12 @@ parseLeaf(
                 }
                 else if (value.isInt())
                 {
-                    if (value.asInt() <
-                            std::numeric_limits<std::int32_t>::min() ||
+                    static_assert(
+                        std::numeric_limits<decltype(value.asInt())>::min() ==
+                        std::numeric_limits<std::int32_t>::min());
+                    static_assert(
+                        std::numeric_limits<decltype(value.asInt())>::max() ==
+                        std::numeric_limits<std::int32_t>::max());
```

Comment:
oops I meant to delete the whole section below. The github "suggestion" feature caught me out.

Reply:
No problem - fixed it in the followup commit